source/operations/cert-manager.rst

```diff
@@ -0,0 +1,108 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
+--------------------------------------------
+
+This guide shows you how to install `cert-manager <https://cert-manager.io>`__ and set up the global ``Cluster Issuer``.
+
+cert-manager manages certificates within Kubernetes clusters.
+The MinIO Operator supports using cert-manager for certificates as an alternative to the MinIO Operator managing certificates for itself and its tenants.
+
+cert-manager obtains valid certificates from an ``Issuer`` or ``ClusterIssuer`` and can automatically renew certificates prior to expiration.
+
+A ``ClusterIssuer`` issues certificates for multiple namespaces.
+An ``Issuer`` only mints certificates for its own namespace.
+
+The following graphic depicts how various namespaces make use of either an ``Issuer`` or ``ClusterIssuer`` type.
+
+- cert-manager is installed in the ``cert-manager`` namespace, which does not have either an ``issuer`` or a ``ClusterIssuer``.
```

Comment:
I'm confused by the `cert-manager` namespace in the diagram, is `minio-operator:issuer` not an `issuer`?

Reply:
cert-manager is the ClusterIssuer.
It can issue certificates to MinIO Operator, tenants, and any other apps that might also run in the cluster, whether related or unrelated to MinIO.

Reply:
We can address that with other comments on the graphic.
It should not have either an Issuer or ClusterIssuer in that namespace. But it operates separately from the other namespaces.

Reply:
See suggestion in other thread here
https://github.com/minio/docs/pull/1317#discussion_r1765237318